src/AlgebraicGeometry/Schemes/AffineVariety/Objects/Methods.jl

```diff
@@ -49,7 +49,7 @@ end
 function Base.show(io::IO, X::AffineVariety)
   io = pretty(io)
   if get(io, :supercompact, false)
-    print(io, "Scheme")
+    print(io, "Affine variety")
```

Comment:
Below this is referred to once as "Affine open subset of ..." and once as "Affine space of dimension ..."